staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -414,9 +414,9 @@ func NewDefaultAPIGroupInfo(group string, registry *registered.APIRegistrationMa
 		GroupMeta:                    *groupMeta,
 		VersionedResourcesStorageMap: map[string]map[string]rest.Storage{},
 		// TODO unhardcode this.  It was hardcoded before, but we need to re-evaluate
-		OptionsExternalVersion: &schema.GroupVersion{Version: "v1"},
-		Scheme:                 scheme,
-		ParameterCodec:         parameterCodec,
-		NegotiatedSerializer:   codecs,
+		//OptionsExternalVersion: &schema.GroupVersion{Version: "v1"},
```

Comment:
I think we can deprecate this field.

Without commenting it out, the API installation of apisever/v1alpha1 will fail with "no type ExportOptions is registered for v1".